### PR TITLE
Decision record proprietary extensions

### PR DESCRIPTION
**Pull request by saschaisele-zf**
## WHAT

Documents the decision regarding the integration of Azure Blob Storage and future integrations.

## FURTHER NOTES

Related to: #661

### CODE REVIEW

File: docs/development/decision-records/2023-08-24-proprietrary-extensions/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+3. Azure Blob Storage
+
+## Approach
+
+To properly incorporate Azure Blob Storage and enable interoperability, there will be a single dataplane which will support HTTP, AWS S3 and Azure Blob Storage.
+This dataplane is then configured based on which extensions, the user wants to use.
+In the case that one or more extensions are not configured, the dataplane is then to exclude these extensions and continue working regardless.
+This makes the use of AWS S3 and Azure Blob Storage entirely optional.
+
+Going forward, the integration of S3 and Azure Blob Storage will stay at the minimum and only bug fixes will be made after the integration.
```
Comment (paullatzelsperger) [inline docs/development/decision-records/2023-08-24-proprietrary-extensions/README.md:35]:
```suggestion
Going forward, the Tractus-X EDC team will limit its efforts to critical bug fixes in the respective upstream repositories.
```

File: docs/development/decision-records/2023-08-24-proprietrary-extensions/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+As these extensions are proprietary in nature, the Tractus-X EDC Team will add no additional features beyond basic support.
+
+The Sftp Protocol is another contender for integrating a more accessible protocol.
+It is however a low priority at the moment and will move on to be an issue for PI 11.
```
Comment (paullatzelsperger) [inline docs/development/decision-records/2023-08-24-proprietrary-extensions/README.md:39]:
```suggestion
It is however a low priority feature at the moment and will be postponed for future iterations.
```

File: docs/development/decision-records/2023-08-24-proprietrary-extensions/README.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+> Note: Azure Blob Storage will be integrated in tune with Tractus-X EDC version 0.6.0
```
Comment (paullatzelsperger) [inline docs/development/decision-records/2023-08-24-proprietrary-extensions/README.md:41]:
```suggestion
> Note: Azure Blob Storage will be integrated with Tractus-X EDC version 0.6.0
```

**Review by paullatzelsperger — COMMENTED** [review]
DR is OK content-wise, only my inner Monk reported a few semantic quirks.

**Comment by saschaisele-zf** [comment]
Always trust the inner monk.

**Review by paullatzelsperger — APPROVED** [review]

**Pull request merged**